product_docs/docs/pgd/5/ddl/ddl-replication-options.mdx

```diff
@@ -5,18 +5,19 @@ navTitle: Options
 
 The [`bdr.ddl_replication`](/pgd/latest/reference/pgd-settings#bdrddl_replication) parameter specifies replication behavior.
 
-`bdr.ddl_replication = all` is the default and replicates DDL to the
+`bdr.ddl_replication = all` is the default. This setting replicates DDL to the
 default replication set, which by default means all nodes. Nondefault
 replication sets don't replicate DDL unless they have a
 [DDL filter](../repsets)
 defined for them.
 
 You can also replicate DDL to specific replication sets using the
-function [`bdr.replicate_ddl_command()`](/pgd/latest/reference/functions#bdrreplicate_ddl_command). This can be helpful if you
-want to run DDL commands when a node is down or if you want to have
+function [`bdr.replicate_ddl_command()`](/pgd/latest/reference/functions#bdrreplicate_ddl_command). This function can be helpful if you
+want to run DDL commands when a node is down. It's also helpful if you want to have
 indexes or partitions that exist on a subset of nodes or rep sets,
```

Comment:
Would the word "include" make more sense than "have" here?

Reply:
It's also helpful if you want to have indexes or partitions that only exist on a subset of nodes or repsets.